bin/ch/Helpers.cpp

```diff
@@ -133,9 +58,9 @@ HRESULT Helpers::LoadScriptFromFile(LPCSTR filename, LPCWSTR& contents, bool* is
     //
     // Read the entire content as a binary block.
     //
-    fread((void*)contentsRaw, sizeof(char), lengthBytes, file);
+    fread(pRawBytes, sizeof(BYTE), lengthBytes, file);
     fclose(file);
-    *(WCHAR*)((byte*)contentsRaw + lengthBytes) = _u('\0'); // Null terminate it. Could be LPCWSTR.
+    *reinterpret_cast<WCHAR*>(pRawBytes + lengthBytes) = 0; // Null terminate it. Could be UTF16
```

Comment:
> With the new code pRawBytes is of type BYTE*, thus it does not need cast. 

Right but that applies to many other things with `reinterpret_cast` (either they are okay or totally wrong)

> What do you have in mind to make it clear?

Although I'm always in favor of C++ style casts, old code is IMO more explicit. Either way, what's done there is nothing but trick**. When using tricks, IMO, it is nice to warn reader.

** i.e. there wouldn't be a reinterpret_cast from char16 string to char str. (vice versa)